attribute-service-api/src/main/proto/org/hypertrace/core/attribute/service/v1/attribute_metadata.proto

```diff
@@ -137,3 +138,39 @@ message AttributeMetadataFilter {
 message Empty {
 }
 
+message AttributeDefinition {
+  oneof value {
+    string span_path = 1;
```

Comment:
not sure about this. The intent is to describe how to extract from span/trace data for simple raw/enriched attributes (since we don't want the caller to have to know how something is defined, this should encompass any attribute type). Could see it being repeated. Does it need to differentiate for trace? understand enriched vs raw?